assets/js/components/settings/SettingsRenderer.js

```diff
@@ -26,13 +26,26 @@ import { useEffect } from '@wordpress/element';
  */
 import Data from 'googlesitekit-data';
 import { STORE_NAME as CORE_MODULES } from '../../googlesitekit/modules/datastore/constants';
+import DefaultSettingsSetupIncomplete from './DefaultSettingsSetupIncomplete';
 const { useSelect, useDispatch } = Data;
 
 export default function SettingsRenderer( { slug, isOpen, isEditing } ) {
 	const storeName = useSelect( ( select ) => select( CORE_MODULES ).getModuleStoreName( slug ) );
 	const isDoingSubmitChanges = useSelect( ( select ) => select( CORE_MODULES ).isDoingSubmitChanges( slug ) );
 	const haveSettingsChanged = useSelect( ( select ) => select( storeName )?.haveSettingsChanged?.() || false );
-	const { SettingsEditComponent, SettingsViewComponent } = useSelect( ( select ) => select( CORE_MODULES ).getModule( slug ) || {} );
+	const {
+		SettingsEditComponent,
+		SettingsViewComponent,
+		SettingsSetupIncompleteComponent,
+		doesModuleExist,
+		connected,
+	} = useSelect( ( select ) => {
+		const module = select( CORE_MODULES ).getModule( slug );
+		return {
+			...module,
+			doesModuleExist: !! module,
```

Comment:
I like the idea to return this as a named value but the name is not quite accurate.
A module that doesn't exist here could be either a module that hasn't been registered/loaded yet, or a module that really doesn't exist (will not be registered).

In this case I think it would be more accurate for our use to name this something like `moduleLoaded` instead.
```suggestion
			moduleLoaded: !! module,
```